docker-compose.yml

```diff
@@ -1,93 +1,91 @@
+# On-Site Version 4.0
 version: "3.6"
 
 x-labels: &k8s-labels
   kompose.image-pull-secret: pullsecret
 
 services:
+  csd-platform:
+    image: ccdcrepository.azurecr.io/onsite/ccdc-web-platform:4.0.0
+    restart: unless-stopped
+    ports:
+      - ${PLATFORM_PORT}:8443
+    labels:
+      <<: *k8s-labels
+      kompose.service.type: LoadBalancer
+      kompose.service.expose: "true"
+    depends_on:
+      - ccdc-identity
+      - webcsd
+      - csd-python-gateway
+
+  csd-python-gateway:
+    environment:
+      - PlatformSecurity__DisableSSLValidation=true
```

Comment:
As discussed, adding a comment here so it's not missed: 
Should this be set to true by default? If a user wants to use SSL validation, we should set this to false as this isn't recommended for prod usage.